bedrock/externalpages/templates/externalpages/pocket/pocket-hits-signup.html

```diff
@@ -0,0 +1,61 @@
+    <h3 class="pocket-signup-hits-section-header publisher">See stories from publishers like these</h3>
+    <ul class="publisher-list">
+        <li class="publisher-list-item"><img src="{{ static('img/externalpages/pocket/pocket-hits-signup/popular-science.png') }}" alt="Popular Science" /> </li>
+        <li class="publisher-list-item"><img src="{{ static('img/externalpages/pocket/pocket-hits-signup/city-lab.png') }}" alt="Citylab"/></li>
+        <li class="publisher-list-item"><img src="{{ static('img/externalpages/pocket/pocket-hits-signup/nautilus.png') }}" alt="Nautilus"/></li>
+        <li class="publisher-list-item"><img src="{{ static('img/externalpages/pocket/pocket-hits-signup/outside.png') }}" alt="Outside"/></li>
+        <li class="publisher-list-item"><img src="{{ static('img/externalpages/pocket/pocket-hits-signup/atlantic.png') }}" alt="The Atlantic"/></li>
+        <li class="publisher-list-item"><img src="{{ static('img/externalpages/pocket/pocket-hits-signup/mental-floss.png') }}" alt="Mental Floss"/></li>
+        <li class="publisher-list-item"><img src="{{ static('img/externalpages/pocket/pocket-hits-signup/texas-monthly.png') }}" alt="Texas Monthly"/></li>
+        <li class="publisher-list-item"><img src="{{ static('img/externalpages/pocket/pocket-hits-signup/saveur.png') }}" alt="Saveur"/></li>
+        <li class="publisher-list-item"><img src="{{ static('img/externalpages/pocket/pocket-hits-signup/narratively.png') }}" alt="Narratively"/></li>
+        <li class="publisher-list-item"><img src="{{ static('img/externalpages/pocket/pocket-hits-signup/atlas-obscura.png') }}" alt="Atlas Obscura"/></li>
+        <li class="publisher-list-item"><img src="{{ static('img/externalpages/pocket/pocket-hits-signup/harvard-review.png') }}" alt="Harvard Business Review"/></li>
+        <li class="publisher-list-item"><img src="{{ static('img/externalpages/pocket/pocket-hits-signup/kiplinger.png') }}" alt="Kiplinger"/></li>
+        <li class="publisher-list-item"><img src="{{ static('img/externalpages/pocket/pocket-hits-signup/fast-company.png') }}" alt="Fast Company"/></li>
+        <li class="publisher-list-item"><img src="{{ static('img/externalpages/pocket/pocket-hits-signup/inc.png') }}" alt="Inc."/></li>
+        <li class="publisher-list-item"><img src="{{ static('img/externalpages/pocket/pocket-hits-signup/quartz.png') }}" alt="Quartz"/></li>
+        <li class="publisher-list-item"><img src="{{ static('img/externalpages/pocket/pocket-hits-signup/guardian.png') }}" alt="The Guardian"/></li>
+    </ul>
+    <h3 class="pocket-signup-hits-section-header signup">Sign up for Pocket Hits</h3>
+     <form class="pocket-hits-signup-form" autocomplete="off">
+        <label for="email-signup-input-bottom">Your email address</label>
+        <input type="email" placeholder="yourname@example.com" id="email-signup-input-bottom" name="top-email-signup-input" value=""/>
```

Comment:
**suggestion(also discussion if needed):** the `name` attribute here shares the same `id` as the above input field at the top of the page. I know that we can use the same `name` attribute for multiple forms if they are grouped together or for the same submission/data collection. In this case for submitting a user's email for the same exact reason, how about we use a more generic name (i.e. `pocket-hits-email-submission`) and also include it for the above input field, that way, if we have to use JavaScript or something that retrieves the emails, we would be gathering data from the two fields since they share the same `name` attribute.